.github/workflows/python-package-conda.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         echo $CONDA/bin >> $GITHUB_PATH
     - name: Install dependencies
       run: |
-        conda env update --file environment.yml --name base
+        conda env update --file environment.yml --name base --solver classic
```

Comment:
I think `--solver classic` needs to be added to line 32, i.e. `conda install flake8 --solver classic`